### ISSUE TITLE
package cannot be installed on php8

### ISSUE THREAD

**Issue by staabm** [idiosyncratic-code/editorconfig-php#16]
```
$ composer install
Loading composer repositories with package information
Warning from https://repo.packagist.org: You are using an outdated version of Composer. Composer 2 is now available and you should upgrade. See https://getcomposer.org/2
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - idiosyncratic/devtools 0.2.0 requires idiosyncratic/coding-standard ^1.0 -> satisfiable by idiosyncratic/coding-standard[1.0.0, 1.0.1, 1.0.2].
    - idiosyncratic/devtools 0.2.1 requires idiosyncratic/coding-standard ^1.0 -> satisfiable by idiosyncratic/coding-standard[1.0.0, 1.0.1, 1.0.2].
    - idiosyncratic/devtools 0.2.2 requires idiosyncratic/coding-standard ^1.0 -> satisfiable by idiosyncratic/coding-standard[1.0.0, 1.0.1, 1.0.2].
    - idiosyncratic/coding-standard 1.0.2 requires php ^7.3 -> your PHP version (8.0.6) does not satisfy that requirement.
    - idiosyncratic/coding-standard 1.0.1 requires php ^7.3 -> your PHP version (8.0.6) does not satisfy that requirement.
    - idiosyncratic/coding-standard 1.0.0 requires php ^7.3 -> your PHP version (8.0.6) does not satisfy that requirement.
    - Installation request for idiosyncratic/devtools ^0.2 -> satisfiable by idiosyncratic/devtools[0.2.0, 0.2.1, 0.2.2].

```

**Comment by jcsilkey** [comment]
This should be resolved with version 0.1.2, can you verify?

**Comment by staabm** [comment]
works, thx
